modules/gdscript/tests/scripts/parser/warnings/static_called_on_instance.gd

```diff
@@ -0,0 +1,11 @@
+class Player:
+	var x = 3
+
+func test():
+	# These should not emit a warning.
+	var _player = Player.new()
```

Comment:
LSAN reports a leak, I suppose it's this one.

Reply:
It doesn't go away if I comment out this line of code or switch to:
```
var node = Node.new()
node.free()